torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -0,0 +1,93 @@
+#include "torch/csrc/jit/passes/constant_propagation.h"
+#include "torch/csrc/jit/passes/dead_code_elimination.h"
+#include "torch/csrc/utils/functional.h"
+#include "torch/csrc/jit/interpreter.h"
+
+namespace torch { namespace jit {
+
+std::unordered_set<Symbol> skip_list = {
+  //FIXME If & Loop require special casing because they cannot be run as a
+  //single node.
+  prim::If,
+  prim::Loop,
+  //FIXME Same problem as in DCE - cpp & python PythonOp and CppOp should be
+  //FIXME treated as having side effects but ONNX depends on them being removed
+  prim::Print,
+ };
```

Comment:
Extra things that need to be skipped: random functions

Reply:
Good point!